### DISCUSSION TITLE
Option to auto-center fixed height windows

### DISCUSSION THREAD

**Discussion by Supreeeme** [YaLTeR/niri#175] [Ideas]
Sometimes windows will have a fixed height when they are created - I've seen this with cagebreak and mpv - and as such they are positioned at the top of the display. I think it would be useful, either as an option or by default, to allow automatically centering these windows vertically.

**Comment by YaLTeR** [comment]
Hmm, yeah, that does make sense.

**Comment by laperex** [comment]
is there any workaround to achieve this effect at least for terminals ?
